### PR TITLE
"Scrappy" website / Conversion to "Showcase"

### PR DESCRIPTION
**Pull request by didoo**
### :pushpin: Summary

This is a very large refactoring of the components' "showcase" website. Since it's going to be used mainly internally (for reference, development and VRT testing) I've maximised the ergonomics for the developers, but I have also tried to keep a certain consistency with the Helios website (fonts, colors, etc) in case in the future we want to re-integrate its content back in the main website.

👉 👉 👉 **Preview**: https://hds-components-git-post-launch-scrappy-website-6b884c-hashicorp.vercel.app/

### :hammer_and_wrench: Detailed description

Here's a high-level list of things I've done (for more details you can look at the commits descriptions):
- removed the old content in the "scrappy" website that was not needed anymore
- removed the "showcase" content in the markdown files and added a placeholder text
- reworked the page layout, responsiveness, overall styling
- added a bunch of helper components (or ported existing ones) that are used for the showcase composition:
  - `Shw::Divider`
  - `Shw::Outliner`
  - `Shw::Placeholder`
  - `Shw::Text` (with all the sub-variants `Shw::Text::H1/2/3/4` and `Shw::Text::Body`)
  - `Shw::Flex`
  - `Shw::Grid`
  - `Shw::Label`
- refactored all the showcase pages using the new components (and entirely reviewed and standardized the page content and its organization)

### :link: External links

Jira ticket: https://hashicorp.atlassian.net/browse/HDS-1502

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable 

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: packages/components/tests/dummy/app/styles/showcase-pages/tabs.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// TAB
+
+body.components-tabs {
+  .shw-component-tabs-layout-highlight {
+    .hds-tabs__tab {
+      outline: 1px dashed #78909c;
```
Comment (KristinLBradley) [inline packages/components/tests/dummy/app/styles/showcase-pages/tabs.scss:6]: Currently the outline is getting cut off. You could add this style to fix:
`outline-offset: -1px;`

**Review by alex-ju — APPROVED** [review]
Great diff! 📉 Haven't spotted anything off aside from a missing import for `ember-power-select`, but I can follow up on that. Also, there would likely be some missing headers, but it's a good opportunity to see the automation in action, so I wouldn't stress about them.

File: packages/components/tests/dummy/app/styles/showcase-components/sample/grid.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+// DEBUG
+
+// .shw-sample-grid {
+//   outline: 4px dotted blue;
+// }
+// .shw-sample-grid__item {
+//   outline: 1px solid green;
+// }
```
[the remark this answers is not in the document]
Comment (didoo) [reply packages/components/tests/dummy/app/styles/showcase-components/sample/grid.scss]: yes, I found the very useful when developing the pages.

File: packages/components/tests/dummy/app/styles/showcase-pages/tabs.scss (hunk 1 above)
Comment (didoo) [reply packages/components/tests/dummy/app/styles/showcase-pages/tabs.scss:6]: Good point! Fixed it :)

File: packages/components/tests/dummy/app/styles/showcase-pages/power-select.scss [hunk 3]
```diff
@@ -1,11 +0,0 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: MPL-2.0
- */
-
-// POWER-SELECT
-@import "ember-power-select";
```
[the remark this answers is not in the document]
Comment (didoo) [reply packages/components/tests/dummy/app/styles/showcase-pages/power-select.scss]: @alex-ju fixed here:
https://github.com/hashicorp/design-system/pull/1170/commits/8bac3af442910b27a673f0219f7459ba15b806b4

Can you please check if now it's OK?
https://hds-components-git-post-launch-scrappy-website-6b884c-hashicorp.vercel.app/overrides/power-select

[the remark this answers is not in the document]
Comment (alex-ju) [reply packages/components/tests/dummy/app/styles/showcase-pages/power-select.scss]: All good now – thank you!

**Pull request merged**